tests/components/emulated_hue/test_upnp.py

```diff
@@ -20,44 +20,15 @@
 JSON_HEADERS = {CONTENT_TYPE: const.CONTENT_TYPE_JSON}
 
 
-def setup_hass_instance(emulated_hue_config):
```

Comment:
I had to get rid of https://github.com/home-assistant/home-assistant/pull/17270/files#diff-27b1be538e8e4042d1123b15f2bda51dL57 since it was actually saving the restore_states file in the testing config, and isn't necessary anymore anyway. When I opened this file I realized the setup methods were duplicated here, presumably leftover from a refactoring. This one was unused.